corehq/apps/enterprise/views.py

```diff
@@ -84,7 +84,32 @@ def enterprise_dashboard(request, domain):
         'current_page': {
             'page_name': _('Enterprise Dashboard'),
             'title': _('Enterprise Dashboard'),
-        }
+        },
+        'header_title': request.account.name,
+        'metric_type': 'Enterprise Dashboard',
+    }
+    return render(request, "enterprise/enterprise_dashboard.html", context)
+
+
+@use_tempusdominus
+@use_bootstrap5
+@always_allow_project_access
+@require_enterprise_admin
+@login_and_domain_required
+@toggles.ENTERPRISE_DASHBOARD_IMPROVEMENTS.required_decorator()
+def security_watchtower(request, domain):
+    if not has_privilege(request, privileges.PROJECT_ACCESS):
+        return HttpResponseRedirect(reverse(EnterpriseBillingStatementsView.urlname, args=(domain,)))
+
+    context = {
```

Comment:
would be good to use https://github.com/dimagi/commcare-hq/blob/8da6e78d52137f5d9cdbff051d09f9cdbe940952/corehq/apps/hqwebapp/context.py#L7

Reply:
use `page_title` and `page_name`

`page_name` is "Security Watchtower for <request.account.name>"

and then specify a `Section` so the breadcrumbs work properly

Reply:
Addressed in 54bb2aa540fe0e38f0036882a6fe2d034d40c7f0